kivymd/uix/snackbar.py

```diff
@@ -309,7 +316,9 @@ class BaseSnackbar(MDCard):
     All Snackbars will be made off of this `BaseSnackbar`. `BaseSnackbar` will always try to fill the remainder of the screen with your Snackbar.
     To make your Snackbar dynamic and symetric with snackbar_x. Set size_hint_x like below:
```

Comment:
Use PEP8 - if possible, do not exceed 79 characters in length.

Reply:
Fixing it right now! I will make sure to review my comments for PEP compliance.